docs/variants/msi_z690/montage.md

```diff
@@ -0,0 +1,123 @@
+
+1. To install the CPU cooler, the parts shown in the picture below are
+    necessary:
+
+    ![Items](images/msi_z690_montage_coller_items.jpg)
+
+1. Attach the black stand to the bottom of the motherboard, then apply and screw
+    down the remaining parts except the cooler itself and apply thermal paste.
+    After these steps, the CPU area should look like this:
+
+    ![CPU Cooler](images/msi_z690_montage_thermal_paste.jpg)
+
+1. Then attach the cooler from above by tightening the two screws. Most likely,
+    for this operation, it is necessary to dismantle the fan and restore it
+    after tightening the screws.
+
+    ![CPU Cooler 2](images/msi_z690_montage_coller.jpg)
+
+1. Install the memory module into the DIMM slots. To insert a single module,
+    open the latch on both sides. Then insert the RAM and gently press the
+    module downward at both ends of the module and the latch will close
+    automatically. Always insert memory modules in the DIMMA2 slot first.
+
+    ![Installed RAM](images/msi_z690_montage_ram.jpg)
+
+1. Install the M.2 solid-state drive (SSD) into the M.2 slot. Remove the two
+    screws from the cover and the one responsible for disk stabilization. Insert
+    the SSD into the M.2 slot and tighten the previously removed screws.
+
+    ![Install Storage](images/msi_z690_montage_storage.jpg)
+
+1. Take the enclosure of the computer and open one side as shown in the picture
+    below:
+
+    ![Open enclosure](images/msi_z690_montage_enclosure.jpg)
+
+1. Install the rear panel by inserting it from the inside of the enclosure
+    and pressing it lightly.
+
+    ![Rear panel](images/msi_z690_montage_rear_panel.jpg)
+
+1. Insert the motherboard into the case but do not twist it, some pins are
+    hard to connect when the motherboard is screwed. The photo below shows the
+    slots that should be filled in the next steps.
+
+    ![Slots](images/msi_z690_montage_slots.png)
+
+1. Connect the cables from the front panel to the appropriately marked places:
+
+    ![Front Panel](images/msi_z690_montage_front_panel.png)
+
+    A detailed description of the connection cable marked `B` can be found in
+    the [Board manual](https://download.msi.com/archive/mnu_exe/mb/PROZ690-AWIFIDDR4_PROZ690-ADDR4100x150.pdf)
+    in section: JFP1, JFP2: Front Panel Connectors.
+
+1. Connect the speaker to the slot marked before `E`:
+
+    ![Speaker](images/msi_z690_montage_speaker.jpg)
+
+1. Now is the time to screw the motherboard onto the case.
+
+1. Connect the power supply to the motherboard using the cables marked RE25 for
+    connection to the `G` and `H` slots and the cable marked RJ21 to connect the
+    `F` slots.
+
+    ![Power Supply](images/msi_z690_montage_power_supply.png)
+
+1. Connect the fan built into the enclosure to the indicated place:
+
+    ![Fan](images/msi_z690_montage_fan.jpg)
+
+1. Close the case and connect the computer to power. Start it with the button
```

Comment:
This section is so long. I think, that we should split it into several subsections, i. e. one section per one element.

Reply:
I splitted this section